modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItJoinTest.java

```diff
@@ -843,6 +1067,7 @@ private static Stream<Arguments> joinTypes() {
         Stream<Arguments> types = Arrays.stream(JoinType.values())
                 // TODO: https://issues.apache.org/jira/browse/IGNITE-21286 remove filter below
                 .filter(type -> type != JoinType.CORRELATED)
+                .filter(type -> type != JoinType.HASHJOIN)
```

Comment:
Why there is need to exclude HASH_JOIN? It all tests except for `testIsNotDistictFrom` pass.

Reply:
let's add TODO referencing a ticket with support of IS NOT DISTINCT operator

Reply:
done, thanks